### PR TITLE
Behavior line chart

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard10**
Draft PR containing the behavior line chart feature.

Todo:
- wait for #883 to be merged to the feature base, then fix all merge conflicts. The PR will become much smaller when these changes in both branches get merged.
- copy-paste `multipleLineChart.tsx` to something like `behaviorLineChart.tsx` and revert all changes to `multipleLineChart.tsx`. They are too different and currently resulted in too many if/else statements.
- fix UI to match designs (e.g. rendering introduction-text and behavior-select next to each other)

### CODE REVIEW

File: src/components-styled/select.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      ))}
+    </InstallationSelect>
+  );
+}
+
+const InstallationSelect = styled.select(
+  css({
+    display: 'inline-block',
+    minWidth: '15em',
+    border: '1px solid lightGray',
```
Comment (VWSCoronaDashboard4) [inline src/components-styled/select.tsx:46]: This results in the wrong color;

`borderColor: 'lightGray'` is needed to trigger the correct styled system color.

File: src/components-styled/select.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    appearance: 'none',
+    padding: 2,
```
Comment (VWSCoronaDashboard4) [inline src/components-styled/select.tsx:48]: Consider using shorthands, `p` for padding, `bg` for background

File: src/components-styled/select.tsx (hunk 1 above)
Comment (VWSCoronaDashboard10) [reply src/components-styled/select.tsx]: good catch!

File: src/style/theme.ts [hunk 3]
```diff
@@ -63,11 +63,11 @@ const breakpoints: Partial<Breakpoints> = [
   '100em',
 ];
 
-breakpoints.xs = breakpoints[0];
-breakpoints.sm = breakpoints[1];
-breakpoints.md = breakpoints[2];
-breakpoints.lg = breakpoints[3];
-breakpoints.xl = breakpoints[4];
+breakpoints.xs = breakpoints[0]; // ~420px
```
Comment (VWSCoronaDashboard4) [inline src/style/theme.ts:66]: :100:

**Review by VWSCoronaDashboard4 — APPROVED** [review]

**Pull request merged**